PhysicsTools/PatAlgos/test/patTuple_addBTagging_cfg.py

```diff
@@ -126,18 +137,19 @@
 process.patJetsAK8PFCHS.addTagInfos = True
 
 # uncomment the following lines to add subjets of ak8PFJetsCHSSoftDrop with new b-tags to your PAT output
+from pdb import set_trace
 addJetCollection(
    process,
    labelName = 'AK8PFCHSSoftDropSubjets',
    jetSource = cms.InputTag('ak8PFJetsCHSSoftDrop','SubJets'),
    jetCorrections = ('AK4PFchs', cms.vstring(['L1FastJet', 'L2Relative', 'L3Absolute']), 'Type-2'), # Using AK4 JECs for subjets which might not be completely appropriate
    algo = 'AK',  # needed for subjet flavor clustering
-   rParam = 0.8, # needed for subjet flavor clustering
    btagDiscriminators = btagDiscriminators,
    explicitJTA = True,  # needed for subjet b tagging
    svClustering = True, # needed for subjet b tagging
    fatJets = cms.InputTag("ak8PFJetsCHS"),               # needed for subjet flavor clustering
-   groomedFatJets = cms.InputTag("ak8PFJetsCHSSoftDrop") # needed for subjet flavor clustering
+   groomedFatJets = cms.InputTag("ak8PFJetsCHSSoftDrop"), # needed for subjet flavor clustering
+   rParam = 0.8, # needed for subjet flavor clustering
```

Comment:
This was actually pretty useful for debugging an error. If you issue "step" to pdb on the previous configuration it actually enters the InputTag creation rather then the pat sequence modifier, hence I left it there in case we will need debugging of the sequence in future.